src/models/utils.js

```diff
@@ -230,10 +230,10 @@ const wrapper = (callback) => (params) =>
     log.warn('error retrieving server props:', e);
     if (e.response?.status === 404) return { notFound: true };
     return {
-      redirect: {
-        destination: '/500',
-        permanent: false,
-      },
+      // redirect: {
+      //   destination: '/500',
+      //   permanent: false,
+      // },
```

Comment:
I think commenting out code is not a good practice, to keep the code clean,  let's simply remove it.

Reply:
I removed the commented code, thanks for the suggestion.